### PR TITLE
stbtrace cleanup

### PR DESCRIPTION
**Pull request by brad-lewis**
Added top level stbtrace directory
Ran python code through flake8 (complains more than pep8)

### CODE REVIEW

**Comment by jgallag88** [comment]
It looks like the directory structure here moves in the opposite direction from what @prashks has in https://github.com/delphix/performance-diagnostics/pull/4. I had also thought we were going to use the style that @prashks has. Can we keep the layout for the `sbtrace` files they way they are, or do you feel that this layout is preferable?

**Comment by prashks** [comment]
Agree with @jgallag88, this is moving it in opposite direction, we want to move towards common layout between `estat` and `stbtrace`, @brad-lewis please see my PR https://github.com/delphix/performance-diagnostics/pull/4

**Comment by brad-lewis** [comment]
Updated to jive with PR #4.

**Comment by jgallag88** [comment]
@brad-lewis The flake8 action reports a few more lint errors. Were those unexpected, or are these intentionally being left for some follow-up change?

File: cmd/estat [hunk 1]
```diff
@@ -25,9 +25,8 @@ from bcc import BPF
 import getopt
 from glob import glob
 import os
-import subprocess
 import sys
-from time import sleep, time, strftime
+from time import sleep, strftime
 
 #
 # Find BCCHelper. If we are being run from the repo, we should be able to find
```
Comment (sebroy) [inline cmd/estat]: This comment looks incomplete, can you fix that while you're here?

Comment (jgallag88) [reply cmd/estat]: This looks complete to me. Can you say what's missing? Or maybe it's worded a bit awkwardly?

Comment (sebroy) [reply cmd/estat]: There's no subject in that last sentence, it's not a syntactically valid English sentence.

Comment (jgallag88) [reply cmd/estat]: I don't see the issue, but maybe I'm just a bit slow today. Maybe removing use of the imperative mood would be helpful?
> If we can't find that directory, we should look for BCCHelper in its install location.

Comment (sebroy) [reply cmd/estat]: Oh, I see what the comment means to say now that you reword it that way. Based on the wording, I thought it was implying that if we aren't running from the repo, then we wouldn't be able to find (whatever it is we're finding).

Comment (brad-lewis) [reply cmd/estat]: How about? 
# We need to find BCCHelper.  If being run from the repo, we should find
 # it in the repo's lib/ directory.  If we can't find that, look for BCCHelper
 # in its install location.

File: lib/iterative_template.py [hunk 2]
```diff
@@ -49,15 +49,15 @@ class IterativeTemplate:
     # helper.add_am.group(2)("$hist.name$", BCCHelper.$hist.aggtype$)
     # }
     #
-    TOP_ITERATIVE_REGEX = '\$(\w+)\:(\{\w+\|[^}]*})\$'
+    TOP_ITERATIVE_REGEX = '\$(\w+)\:(\{\w+\|[^}]*})\$'  # noqa: W605
```
[the remark this answers is not in the document]
Comment (brad-lewis) [reply lib/iterative_template.py]: Didn't know about raw strings.  Thanks.

File: cmd/estat (hunk 1 above)
Comment (brad-lewis) [reply cmd/estat]: I fixed the flake8 errors that came in during the rebase.

Comment (sebroy) [reply cmd/estat]: Sounds great!

Comment (jgallag88) [reply cmd/estat]: nit: 'If we can't find that,' might read a bit better as 'If we can't find that _directory_,'. Other than that, sounds good.

**Comment by brad-lewis** [comment]
Passing the lint checks now and with the agreed upon comment text.

**Pull request merged**